app/stream.go

```diff
@@ -0,0 +1,169 @@
+package app
```

Comment:
I believe you're the first to write real Golang streaming code 😆 . Exciting! This is looking really good; I have some small comments and I can help a bit with the testing. Why don't we sync offline about it. @jckras do know if any of the data client APIs are streaming? Or is the `TailRobotPartLogs` the only one we'll have to deal with?

Reply:
I am working on two streaming wrappers in the data API client - `FileUpload()` and `StreamingDataCapture()` , hoping to get a PR up by the end of the day

Reply:
There's `board.StreamTicks()` but it doesn't seem to test what's in the actual channel.